src/components/CookiesBanner/index.tsx

```diff
@@ -97,14 +98,21 @@ interface CookiesBannerFormProps {
 const CookiesBanner = (): ReactElement => {
   const classes = useStyles()
   const dispatch = useRef(useDispatch())
-
+  const history = useHistory()
   const [showAnalytics, setShowAnalytics] = useState(false)
   const [showIntercom, setShowIntercom] = useState(false)
   const [localNecessary, setLocalNecessary] = useState(true)
   const [localAnalytics, setLocalAnalytics] = useState(false)
   const [localIntercom, setLocalIntercom] = useState(false)
 
   const showBanner = useSelector(cookieBannerOpen)
+  const safeAppView = history.location.search.substr(0, 7) === '?appUrl'
```

Comment:
I'd suggest doing it like the Apps route itself is doing it.
https://github.com/gnosis/safe-react/blob/development/src/routes/safe/components/Apps/index.tsx#L8-L14
I would extract the logic to grab the `appUrl` into a function in the Apps [utils](https://github.com/gnosis/safe-react/blob/development/src/routes/safe/components/Apps/utils.ts).

Reply:
I see, useLocation must be used inside a react component or hook, do you think i should create a hook to do this?

Reply:
useLocation is already a hook